src/components/RocketsComponents.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getRocketData } from './Redux/Rockets/rocketsSlice';
+import CancelReservationCTA from './CancelReservationCTA';
+import ReserveRocketCTA from './ReserveRocketCTA';
+import RocketReservedCTA from './RocketReservedCTA';
+import './Rockets.css';
+import './RocketCTA.css';
+
+function RocketsComponents() {
+  const { rocketArr } = useSelector((store) => store.rockets);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getRocketData());
+  }, []);
+  return (
+    <>
```

Comment:
- [ ] Make sure that your application behaves the way it should as per requirement. Your application no doubt works, but once I booked a rocket go to the profile section and then click on the `Rocket` page. It lost its state, and it should not happen as per the requirement. Your data should be fetched for the very first time and then no matter a user moving from one page to another, back and forth. Your state should be preserved within the store. Kudos to the partner who implemented the `Mission` component very well because the `Mission` page work exactly what we want as a Code Reviewer.